docs/pages/functionality/filter-management.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+title: Filter Management
+permalink: functionality/filter-management
+type: functionality
+language: en
+list: include
+---
+
+### What is an Envoy ?
```

Comment:
@sayantan1413 it would be good to start off with an initial description of filter management (could be just a few sentences), and then move into "What is" sections.